V2.1.0 (#204)
hCaptcha/hcaptcha-wordpress-plugin: readme.txt

=== Please note ===

> **NOTE:** This is a community-developed plugin. All integrations were submitted by developers who didn't want to wait for a particular plugin to add native hCaptcha support.

Reviewer: ```suggestion
**NOTE:** This is a community-developed plugin. Your PRs are welcome.
```
